### PR TITLE
Added stellate morphology

### LINKED ISSUE #382: NTR: stellate morphology
**Term:** 
stellate morphology

**Definition:**
A cell morphology that inheres in neurons which have dendritic processes radiating for the cell body that forms a star-like shape
_Note: Need to find ref_

**Parent Class:**
Cell Morphology #364

**Others:**
Stellate Morphology: SAO:9271919883, ILX:0111036

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #382

### CODE REVIEW

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -21847,6 +21847,15 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070010
+name: stellate morphology
+def: "A cell morphology that inheres in neurons which have dendritic processes radiating for the cell body that forms a star-like shape" [ISBN:9780123973481]
```
Comment (patrick-lloyd-ray) [inline src/ontology/pato-edit.obo:21853]: Add full stop in definition.

**Review by patrick-lloyd-ray — COMMENTED** [review]
Small change -- looks good otherwise.

File: src/ontology/pato-edit.obo [hunk 2]
```diff
@@ -21847,6 +21847,15 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070010
+name: stellate morphology
+def: "A cell morphology that inheres in neurons which have dendritic processes radiating for the cell body that forms a star-like shape." [ISBN:9780123973481]
```
Comment (dosumis) [inline src/ontology/pato-edit.obo:21853]: > def: "A cell morphology that inheres in neurons which have dendritic processes radiating for the cell body that forms a star-like shape." [ISBN:9780123973481]

->
def: "A cell morphology that inheres in neurons that have dendritic processes radiating from the cell body forming a star-like shape." [ISBN:9780123973481]

**Review by dosumis — CHANGES_REQUESTED** [review]
Minor text change

**Review by dosumis — APPROVED** [review]

**Review by patrick-lloyd-ray — APPROVED** [review]

**Pull request merged**